app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragment.kt

```diff
@@ -57,14 +60,21 @@ class PlatformParametersFragment : InjectableFragment() {
 
   override fun onSaveInstanceState(outState: Bundle) {
     super.onSaveInstanceState(outState)
-    val platformParameterStates = platformParametersFragmentPresenter.platformParameterStates.map {
-      OverriddenPlatformParameter.newBuilder()
-        .setId(it.key)
-        .setOverriddenValue(it.value)
-        .build()
-    }
+    val validParameterOverrides =
+      platformParametersFragmentPresenter.platformParameterStates.mapNotNull { (key, value) ->
+        value?.let {
+          OverriddenPlatformParameter.newBuilder()
+            .setId(key)
+            .setOverriddenValue(it)
+            .build()
+        }
+      }
+    val invalidParameterIds = platformParametersFragmentPresenter.platformParameterStates
```

Comment:
Making a small adjustment to the suggestion of variable naming for clarity. `invalidInputParameterIds` or `invalidValueParameterIds` - just to be specific that it's the input that's invalid.
```suggestion
    val invalidInputParameterIds = platformParametersFragmentPresenter.platformParameterStates
```

Reply:
updated